src/or/connection.c

```diff
@@ -1162,6 +1167,8 @@ connection_listener_new(const struct sockaddr *listensockaddr,
   tor_addr_t addr;
   int exhaustion = 0;
 
+  if (addr_in_use) *addr_in_use = 0;
```

Comment:
I'm assuming that this formatting problem (and other similar ones) gets fixed in a later commit

Reply:
Yes, in ee3fbf1195052019c73c8c5e43b64bf5f17ab4a2 and 7cd5c0004eb4c55a598e0f918e6861d8293b6d70.